go.sum

```diff
@@ -880,9 +1118,28 @@ google.golang.org/grpc v1.21.1/go.mod h1:oYelfM1adQP15Ek0mdvEgi9Df8B9CZIaU1084ij
 google.golang.org/grpc v1.22.1/go.mod h1:Y5yQAOtifL1yxbo5wqy6BxZv8vAUGQwXBOALyacEbxg=
 google.golang.org/grpc v1.23.0/go.mod h1:Y5yQAOtifL1yxbo5wqy6BxZv8vAUGQwXBOALyacEbxg=
 google.golang.org/grpc v1.23.1/go.mod h1:Y5yQAOtifL1yxbo5wqy6BxZv8vAUGQwXBOALyacEbxg=
+google.golang.org/grpc v1.25.1/go.mod h1:c3i+UQWmh7LiEpx4sFZnkU36qjEYZ0imhYfXVyQciAY=
 google.golang.org/grpc v1.26.0/go.mod h1:qbnxyOmOxrQa7FizSgH+ReBfzJrCY1pSN7KXBS8abTk=
 google.golang.org/grpc v1.27.0/go.mod h1:qbnxyOmOxrQa7FizSgH+ReBfzJrCY1pSN7KXBS8abTk=
 google.golang.org/grpc v1.27.1/go.mod h1:qbnxyOmOxrQa7FizSgH+ReBfzJrCY1pSN7KXBS8abTk=
+google.golang.org/grpc v1.28.0/go.mod h1:rpkK4SK4GF4Ach/+MFLZUBavHOvF2JJB5uozKKal+60=
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS secret access key. ([view](https://sider.review/gh/repos/237181535/pulls/194#issue-6191029))
>
> <details>
>
> AWS secret access key is an important credential that allows you to use AWS programmatically.
> Once this leaks, an attacker can access and operate your AWS resources.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.secret_access_key` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/237181535/pulls/194#issue-6191029) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS secret access key.